{
    "ID": {
        "GitHub_Advisory_Database": {
            "aliases": [
                "CVE-2020-13254"
            ]
        },
        "Nomisec": {
            "id": 270368383,
            "name": "django-cve-2020-13254",
            "full_name": "danpalmer/django-cve-2020-13254"
        }
    },
    "summary": "Data leakage via cache key collision in Django",
    "details": {
        "GitHub_Advisory_Database": "An issue was discovered in Django version 2.2 before 2.2.13 and 3.0 before 3.0.7. In cases where a memcached backend does not perform key validation, passing malformed cache keys could result in a key collision, and potential data leakage."
    },
    "description": {
        "Nomisec": "Vulnerability demonstration for Django CVE-2020-13254"
    },
    "topics": [
        "key",
        "backend",
        "malformed",
        "cache",
        "cases",
        "keys",
        "potential",
        "perform",
        "result",
        "13"
    ],
    "severity": [],
    "affected": [],
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://nvd.nist.gov/vuln/detail/CVE-2020-13254"
        },
        {
            "type": "WEB",
            "url": "https://github.com/django/django/commit/07e59caa02831c4569bbebb9eb773bdd9cb4b206"
        },
        {
            "type": "WEB",
            "url": "https://github.com/django/django/commit/84b2da5552e100ae3294f564f6c862fef8d0e693"
        },
        {
            "type": "WEB",
            "url": "https://docs.djangoproject.com/en/3.0/releases/security"
        },
        {
            "type": "PACKAGE",
            "url": "https://github.com/django/django"
        },
        {
            "type": "WEB",
            "url": "https://groups.google.com/d/msg/django-announce/pPEmb2ot4Fo/X-SMalYSBAAJ"
        },
        {
            "type": "WEB",
            "url": "https://lists.debian.org/debian-lts-announce/2020/06/msg00016.html"
        },
        {
            "type": "WEB",
            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/4A2AP4T7RKPBCLTI2NNQG3T6MINDUUMZ"
        },
        {
            "type": "WEB",
            "url": "https://security.netapp.com/advisory/ntap-20200611-0002"
        },
        {
            "type": "WEB",
            "url": "https://usn.ubuntu.com/4381-1"
        },
        {
            "type": "WEB",
            "url": "https://usn.ubuntu.com/4381-2"
        },
        {
            "type": "WEB",
            "url": "https://www.debian.org/security/2020/dsa-4705"
        },
        {
            "type": "WEB",
            "url": "https://www.djangoproject.com/weblog/2020/jun/03/security-releases"
        },
        {
            "type": "WEB",
            "url": "https://www.oracle.com/security-alerts/cpujan2021.html"
        }
    ],
    "proof_of_concept": {
        "owner": {
            "html_url": "https://github.com/danpalmer"
        },
        "html_url": "https://github.com/danpalmer/django-cve-2020-13254"
    },
    "has_discussions": null,
    "database_specific": {
        "cwe_ids": [
            "CWE-295"
        ],
        "severity": "MODERATE",
        "github_reviewed": true,
        "github_reviewed_at": "2020-06-04T18:32:36Z",
        "nvd_published_at": "2020-06-03T14:15:00Z"
    },
    "Time_Description": {
        "GitHub_Advisory_Database": {
            "modified": "2022-09-08T14:02:23Z",
            "published": "2020-06-05T16:20:44Z",
            "withdrawn": null
        },
        "Nomisec": {
            "created_at": "2020-06-07T16:42:33Z",
            "updated_at": "2024-08-12T20:02:22Z",
            "pushed_at": "2020-06-07T18:07:46Z"
        }
    }
}